### PR TITLE
updates

### PR DESCRIPTION
**Pull request by guilhermebodin**
Added some datasets and tests fro ARIMA

### CODE REVIEW

File: src/models/arima.jl [hunk 1]
```diff
@@ -275,17 +282,17 @@ function unconstrain_hyperparameters!(model::ARIMA{Fl}) where Fl
         update_unconstrained_value!(model, get_ma_name(model, j), unconstrained_ma[j])
     end
 
-    # Unconstraint variance
-    update_unconstrained_value!(model, "sigma2_η", sqrt(get_constrained_value(model, "sigma2_η")))
+    # Unconstrain variance
+    unconstrain_variance(model, "sigma2_η")
```
Comment (raphaelsaavedra) [inline src/models/arima.jl:286]: Should be `unconstrain_variance!`?

File: src/models/common.jl [hunk 2]
```diff
@@ -1,24 +1,16 @@
...
-TODO
-"""
 system(model::StateSpaceModel) = model.system
 
 """
-TODO
+    has_fit_methods(model_type::Type{<:StateSpaceModel})
 
-Verifica se o model tem todas as funções implementadas
+Verify if a certain `StateSpaceModel` has all the methods perform the fit method.
```
Comment (raphaelsaavedra) [inline src/models/common.jl:11]:
```suggestion
Verify if a certain `StateSpaceModel` has the necessary methods to perform the fit.
```

File: src/systems.jl [hunk 3]
```diff
@@ -1,6 +1,40 @@
-abstract type StateSpaceSystem end
+@doc raw"""
+    StateSpaceSystem
 
+abtract type that unifies the definition of state space models matrices such as ``y, Z, d, T, c, R, H, Q`` for
```
Comment (raphaelsaavedra) [inline src/systems.jl:4]:
```suggestion
Abstract type that unifies the definition of state space models matrices such as ``y, Z, d, T, c, R, H, Q`` for 
```

File: src/systems.jl [hunk 4]
```diff
@@ -35,7 +69,30 @@ end
...
+    \begin{aligned}
+        y_{t} &=  Z_{t}\alpha_{t} + d_{t} + \varepsilon_{t} \quad \varepsilon_{t} \sim \mathcal{N}(0, H_{t})\\
+        \alpha_{t+1} &= T_{t}\alpha_{t} + c_{t} + R_{t}\eta_{t} \quad \eta_{t} \sim \mathcal{N}(0, Q_{t})\\
+    \end{aligned}
+\end{gather*}
+```
+
+where:
+
+* ``y_{t}`` is a scala
```
Comment (raphaelsaavedra) [inline src/systems.jl:88]:
```suggestion
* ``y_{t}`` is a scalar
```

File: src/models/arima.jl (hunk 1 above)
Comment (guilhermebodin) [reply src/models/arima.jl:286]: ok, i gonna change this

**Pull request merged**